src/qt/rpcconsole.cpp

```diff
@@ -910,8 +910,14 @@ void RPCConsole::clear(bool keep_prompt)
 
 void RPCConsole::keyPressEvent(QKeyEvent *event)
 {
-    if(windowType() != Qt::Widget && event->key() == Qt::Key_Escape)
-    {
+    const auto close_key =
+#ifdef Q_OS_ANDROID
```

Comment:
I don't like the `#ifdef`. Any way to remap this globally?

Reply:
@icota Could you elaborate? I'm not sure about `Qt::Key_Back` on non-Android platforms.

Reply:
One point is that this should be done globally rather than just in `rpcconsole`. In our case `Qt::Key_Back` and `Qt::Key_Escape` will always serve the same purpose.

It also should not be an either-or because Android phones can have keyboards hooked up to them and ChromeOS can run APKs as well.

I guess my suggestion would be something like a global macro/function that checks if its escape or back button. If this is truly the only place where we need to check for the back button then `if (event->key == Qt::Key_Escape || event->key == Qt::Key_Back)` is also cool.

Reply:
Thanks! [Implemented](https://github.com/bitcoin-core/gui/pull/493#issuecomment-985075240).